### PR TITLE
MEN-4479 configuration example tooltips

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

File: src/js/components/helptips/helptooltips.js [hunk 1]
```diff
@@ -145,3 +151,72 @@ const DeviceSupportTipComponent = ({ docsVersion }) => (
...
+              on: 'Turn on the light permanently',
+              off: 'Turn off the light permanently',
+              heartbeat: 'Enable heartbeat blinking'
+            }}
+            compact
+          />
+          There are other possible values, but we won&apos;t advertise them here. See MEN-4461
+          <a href="http://www.d3noob.org/2020/07/controlling-activity-led-on-raspberry-pi.html" target="_blank" rel="noopener noreferrer">
+            in this blog post
+          </a>{' '}
```
Comment (kacf) [inline src/js/components/helptips/helptooltips.js:209]: This sentence seems strange: "See MEN-4461 in this blog post or ..."?

What about: "See this blog post or ..." (the ticket reference is not important to get across here)

File: src/js/components/helptips/helptooltips.js [hunk 2]
```diff
@@ -145,3 +151,72 @@ const DeviceSupportTipComponent = ({ docsVersion }) => (
...
+  }
+  return (
+    <>
+      <div id="config-timezone-help" className="fadeIn tooltip help" data-tip data-for="config-timezone-tip" data-event="click focus" style={anchor}>
+        <HelpIcon />
+      </div>
+      <ReactTooltip id="config-timezone-tip" globalEventOff="click" place="bottom" type="light" effect="solid" className="react-tooltip">
+        <>
+          To see the effects of applying a configuration to your device you can set one of the below values to modify the behaviour of your Raspberry Pi status
+          LED
```
Comment (kacf) [inline src/js/components/helptips/helptooltips.js:195]: I would say "Raspberry Pi *green* status LED", just to make the distinction clear from the red one right next to it.

**Review by kacf — APPROVED** [review]
The code is lost on me, but the tool tip content looks good, just a few minor adjustments!

File: src/js/components/helptips/helptooltips.js (hunk 2 above)
Comment (mzedel) [reply src/js/components/helptips/helptooltips.js]: good point - should now be changed!

File: src/js/components/helptips/helptooltips.js (hunk 1 above)
Comment (mzedel) [reply src/js/components/helptips/helptooltips.js:209]: thanks for catching that - it was not supposed to be there and is now gone 👌

**Review by michaelatmender — APPROVED** [review]

**Pull request merged**